infrastructure/terraform/app/otel-collector-td/variables.tf

```diff
@@ -29,20 +29,20 @@ variable "sample_nginx_app_target_group_id" {}
 variable "jaeger_app_target_group_id" {}
 
 variable "otlp_fe_app_target_group_id" {}
-    
+
 variable "otlp_be_app_target_group_id" {}
 
 variable "otlp_collector_app_target_group_id" {}
 
 variable "app_image" {
-    description = "The container image to use"
-    default = "otel/opentelemetry-collector:latest"
+  description = "The container image to use"
+  default     = "otel/opentelemetry-collector:latest"
```

Comment:
as the project is not actively maintained, can we pin the version to the current one so it will work until next round of updates?